### ISSUE TITLE
Pods install not working  - Searching for inspections failed - Ruby Problems 

### ISSUE THREAD

**Issue by aakpro** [CocoaPods/CocoaPods#9513]
――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Users/user/.rbenv/versions/2.2.2/bin/pod install
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?

### Stack

```
   CocoaPods : 1.8.4
        Ruby : ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14]
    RubyGems : 2.4.5
        Host : Mac OS X 10.15.2 (19C57)
       Xcode : 11.3 (11C29)
         Git : git version 2.21.0 (Apple Git-122.2)
Ruby lib dir : /Users/user/.rbenv/versions/2.2.2/lib
Repositories : master - git - https://github.com/CocoaPods/Specs.git @ 61bf78db170d72d56828b8b361d590291df92f81
               trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.4.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
# Uncomment the next line to define a global platform for your project
platform :ios, '12.0'

target 'iOS-Challenge' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  pod 'Alamofire'
  # Pods for iOS-Challenge

  target 'iOS-ChallengeTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'iOS-ChallengeUITests' do
    # Pods for testing
  end

end
```

### Error

```
LoadError - library not found for class Digest::MD5 -- digest/md5
/Users/user/.rbenv/versions/2.2.2/lib/ruby/2.2.0/digest.rb:15:in `const_missing'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-core-1.8.4/lib/cocoapods-core/source/metadata.rb:45:in `path_fragment'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-core-1.8.4/lib/cocoapods-core/cdn_source.rb:311:in `pod_shard_fragment'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-core-1.8.4/lib/cocoapods-core/cdn_source.rb:202:in `search'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-core-1.8.4/lib/cocoapods-core/source/aggregate.rb:83:in `block in search'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-core-1.8.4/lib/cocoapods-core/source/aggregate.rb:83:in `select'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-core-1.8.4/lib/cocoapods-core/source/aggregate.rb:83:in `search'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:416:in `create_set_from_sources'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:385:in `find_cached_set'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:360:in `specifications_for_dependency'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:165:in `search_for'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:274:in `block in sort_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:267:in `each'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:267:in `sort_by'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:267:in `sort_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/delegates/specification_provider.rb:53:in `block in sort_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/delegates/specification_provider.rb:70:in `with_no_such_dependency_error_handling'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/delegates/specification_provider.rb:52:in `sort_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:288:in `initial_state'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:210:in `start_resolution'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/resolution.rb:168:in `resolve'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/molinillo-0.6.6/lib/molinillo/resolver.rb:43:in `resolve'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/resolver.rb:94:in `resolve'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer/analyzer.rb:986:in `block in resolve_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:64:in `section'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer/analyzer.rb:984:in `resolve_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer/analyzer.rb:124:in `analyze'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:410:in `analyze'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:234:in `block in resolve_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:64:in `section'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:233:in `resolve_dependencies'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:156:in `install!'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/command/install.rb:52:in `run'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/lib/cocoapods/command.rb:52:in `run'
/Users/user/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/cocoapods-1.8.4/bin/pod:55:in `<top (required)>'
/Users/user/.rbenv/versions/2.2.2/bin/pod:23:in `load'
/Users/user/.rbenv/versions/2.2.2/bin/pod:23:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=library+not+found+for+class+Digest%3A%3AMD5+--+digest%2Fmd5&type=Issues

Searching for inspections failed: #<Pod::UserInterface::InspectorReporter:0x007fd7a9c22c10> does not handle inspector_successfully_recieved_report

**Comment by dnkoutso** [comment]
Fixed probably in 1.9.0 (Beta) can you try it?

**Comment by aakpro** [comment]
Fixed by reinstalling ruby, configuring rbenv!
